### PR TITLE
Install at cmd in startup script and support configuring SSD disk type

### PR DESCRIPTION
**Pull request by rishisv**
* Flag for choosing boot disk type for creating the VM with SSD storage, defaults to SSD
* Installs the at command in startup script as it is a key dependency

### CODE REVIEW

File: action.sh [hunk 1]
```diff
@@ -146,19 +151,21 @@ function start_vm {
...
   startup_script="
+    apt install -y at && \\
     gcloud compute instances add-labels ${VM_ID} --zone=${machine_zone} --labels=gh_ready=0 && \\
     RUNNER_ALLOW_RUNASROOT=1 ./config.sh --url https://github.com/${GITHUB_REPOSITORY} --token ${RUNNER_TOKEN} --labels ${VM_ID} --unattended ${ephemeral_flag} --disableupdate && \\
     ./svc.sh install && \\
     ./svc.sh start && \\
     gcloud compute instances add-labels ${VM_ID} --zone=${machine_zone} --labels=gh_ready=1
     # 3 days represents the max workflow runtime. This will shutdown the instance if everything else fails.
-    echo \"gcloud --quiet compute instances delete ${VM_ID} --zone=${machine_zone}\" | at now + 3 days
+    echo \"gcloud --quiet compute instances delete ${VM_ID} --zone=${machine_zone}\" | at now + 2 hours
```
Comment (ravwojdyla) [inline action.sh:168]: @rishisv can we accept this as a parameter please?

**Review by ravwojdyla — CHANGES_REQUESTED** [review]
@rishisv sorry for late response, this change looks great but we would need to adjust the TTL change.

**Comment by andodet** [comment]
It would be great to have this completed and merged as Ubuntu 22.04 doesn't currently include `at` and it needs to be installed as a separate dependency.

Happy to open a fresher PR (if this one got abandoned) to get it merged.

**Comment by ravwojdyla** [comment]
@andodet please go ahead, PR are welcome.

**Comment by ravwojdyla** [comment]
Give #30 this is now obsolete. Closing.

**Pull request closed without merging**